include/ProgramRepresentation/ProgramPoint.h

```diff
@@ -9,11 +9,12 @@
 class ProgramPoint {
   private:
     DisjointPVAliasSets programVariableAliasSets;
-    Value *returnValue;
     std::list<ProgramPoint *> successors;
 
-    // the name is same as the branch name that shows up in the IR
-    std::string pointName;
+    // this is the first instruction num that the ProgramPoint describes
+    // To find the most current one, iterate until the next Point does not exist
+    // or the next pointName is higher than the instruction number you are
+    int pointName;
```

Comment:
`pointName` is an odd name for what I think is either:
* an index (e.g., into the list in the associated `ProgramBlock`, or
* a unique identifier for this point

Please add documentation explaining what these numbers _mean_. For example, I'm not sure if you intend these to be unique or not.